eng/common/scripts/Prepare-Release.ps1

```diff
@@ -184,6 +184,7 @@ else
   exit 1
 }
 
+Sanitize-ChangeLogEntry -ChangeLogLocation $packageProperties.ChangeLogPath -VersionString $newVersion
```

Comment:
Shouldn't this be removed now and you pass the SantizeEntry to Confirm-ChangeLogEntry?

Reply:
I don't see you passing the SantizeEntry switch in Prepare-Release